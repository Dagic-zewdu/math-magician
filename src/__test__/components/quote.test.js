import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/quote/quote';

describe('should render without problems', () => {
  it('should render the quote page', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
