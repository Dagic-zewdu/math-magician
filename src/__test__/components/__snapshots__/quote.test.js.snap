// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without problems should render the calculator 1`] = `
<div
  className="sc-bdvvtL dUKpQw"
>
  <p>
    This is awesome quote that bright ur day 
  </p>
  <ul
    className="sc-gsDKAQ"
  >
    <li>
      Positive anything is better than negative nothing
    </li>
    <li>
      Miracles happen to those who believe in them 
    </li>
    <li>
      One small positive thought can change your whole day 
    </li>
    <li>
       Believe you can and you’re halfway there
    </li>
    <li>
       Write it on your heart that every day is the best day in the year.
    </li>
  </ul>
</div>
`;

exports[`should render without problems should render the quote page 1`] = `
<div
  className="sc-bdvvtL dUKpQw"
>
  <p>
    This is awesome quote that bright ur day 
  </p>
  <ul
    className="sc-gsDKAQ"
  >
    <li>
      Positive anything is better than negative nothing
    </li>
    <li>
      Miracles happen to those who believe in them 
    </li>
    <li>
      One small positive thought can change your whole day 
    </li>
    <li>
       Believe you can and you’re halfway there
    </li>
    <li>
       Write it on your heart that every day is the best day in the year.
    </li>
  </ul>
</div>
`;
